app/ui/client/lib/notification.js

```diff
@@ -43,7 +43,7 @@ export const KonchatNotification = {
 					canReply: true,
 				});
 
-				const notificationDuration = notification.duration - 0 || getUserPreference(Meteor.userId(), 'desktopNotificationDuration') - 0;
+				const notificationDuration = notification.duration - 0 || 10;
```

Comment:
`notification.duration` will not be sent anymore.. so this can be changed to fixed `10`